tests/test_firebase_stream.py

```diff
@@ -0,0 +1,53 @@
+from unittest.mock import patch
+
+from nio import Signal
```

Comment:
Line 3,4 here are unused imports

Reply:
I couldn't comment on the files themselves for some reason, but unused imports also in:

Line 4 in test_firebase_read.py
Line 4 in test_firebase_insert.py